mewarpx/mewarpx/mwxrun.py

```diff
@@ -25,6 +25,11 @@
 import warnings
 
 from pywarpx import _libwarpx, picmi
+<<<<<<< HEAD
+from mewarpx.util import compute_step
+import ctypes
+=======
+>>>>>>> 82baac8190e3121d2d4309996b9688e9803428fd
```

Comment:
Not sure if this was intentional, but the merge stuff is still here.
```suggestion
from mewarpx.util import compute_step
import ctypes
```